examples/imitation_learning_openbot/utils.py

```diff
@@ -102,14 +103,14 @@ def generate_video(image_dir, video_path, rate, compress=False):
     process.wait()
 
 
-def plot_tracking_performance_spatial(poses_current, poses_desired, plot_path):
+def plot_tracking_performance_spatial(poses_tracked, poses_desired, plot_path):
 
     fig, (ax) = plt.subplots(1, 1)
 
-    ax.plot(poses_current[0,0], poses_current[0,1], marker="^", markersize=12.0, label="Start", color="tab:blue")
+    ax.plot(poses_tracked[0,0], poses_tracked[0,1], marker="^", markersize=12.0, label="Start", color="tab:blue")
     ax.plot(poses_desired[-1,0], poses_desired[-1,1], marker="^", markersize=12.0, label="Goal", color="tab:orange")
     ax.plot(poses_desired[:,0], poses_desired[:,1], marker="o", markersize=8.0, label="Desired trajectory", color="tab:green")
-    ax.plot(poses_current[:,0], poses_current[:,1], marker="x", markersize=3.0, label="Actual trajectory", color="tab:purple")
+    ax.plot(poses_tracked[:,0], poses_tracked[:,1], marker="x", markersize=3.0, label="Actual trajectory", color="tab:purple")
 
     x_0,x_1 = ax.get_xlim()
```

Comment:
It would be cleaner if these plotting functions assumed meters rather than centimeters.